gradle-jdks-excavator-configurations/src/main/resources/install-jdks.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -eux
+
+GRADLE_DIR=$1
+CERTS_DIR=$2
+
+# used by the resolved_symlink below to resolve the path based on the JAVA_VERSION value. e.g. /usr/local/${JAVA_VERSION}
+# shellcheck disable=SC2034
+SYMLINK_PATTERN=$3
+
+# Loading gradle jdk functions
+scripts_dir="$(dirname "$(readlink -f "$0")")"
```

Comment:
we normally use `$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )`, not sure why.

Reply:
I think it's best to use all caps for global variables and lower case for `local` variables, just to not mix them up